src/export/digitalPlanning/model.ts

```diff
@@ -40,50 +72,35 @@ export class DigitalPlanning {
           this.passport.data?.["user.role"][0] === "applicant"
             ? this.getApplicant()
             : this.getApplicantWithAgent(),
-        property: {
-          address: this.getPropertyAddress(),
-          type: {
-            value: this.passport.data?._address?.["planx_value"],
-            description: this.passport.data?._address?.["planx_description"],
-          },
-          boundary: this.getBoundary(),
-          constraints: {
-            planning: [],
-          },
-        },
+        property: this.getProperty(),
         application: {
           type: this.getApplicationType(),
           fee: this.getApplicationFee(),
-          declaration: {
-            accurate: true, // Boolean(this.passport.data?.["application.declaration.accurate"][0]),
-            connection:
-              this.passport.data?.["application.declaration.connection"][0],
-          },
-        },
-        proposal: {
-          projectType: [],
-          description:
-            this.passport.data?.["proposal.description"] ||
-            "Not provided in Prior Approval?",
+          declaration: this.getApplicationDeclaration(),
         },
+        proposal: this.getProposal(),
       },
-      result: [],
+      result: this.getResult(),
+      responses: this.getResponses(),
+      files: this.getFiles(),
       metadata: {
         service: {
-          publishedFlowId: "b3320abe-f5bc-4185-b61f-40e9e65f07ad",
-          name: "Test",
-          owner: "Test",
-          url: "https://www.planx.uk/",
+          flowId: this.metadata.flow.id,
+          publishedFlowId: this.metadata.flow.publishedFlows[0].id,
+          name: capitalize(this.metadata.flow.slug.replaceAll?.("-", " ")),
+          owner: this.metadata.flow.team.slug,
+          url: `https://www.editor.planx.uk/${this.metadata.flow.team.slug}/${this.metadata.flow.slug}/preview`,
         },
         session: {
           source: "PlanX",
-          id: "b3320abe-f5bc-4185-b61f-40e9e65f07ad", // this.sessionId throws 'must match format UUID' error
-          createdAt: "2018-11-13T20:20:39+00:00",
-          submittedAt: "2018-11-13T20:20:39+00:00",
+          id: this.sessionId,
+          createdAt: this.metadata.createdAt,
+          submittedAt: this.metadata.submittedAt,
+        },
+        schema: {
+          url: "https://theopensystemslab.github.io/digital-planning-data-schemas/v0.0.1/schema.json", // @todo populate version based on submittedAt
```

Comment:
Right now, everytime we publish a new schema release, we'll need to update the static schema.json in planx-core too - because we populate some payload values directly from the schema definitions.

So I realised I think this is as simple as a string template that let's us still keep the URL type here:
```https://theopensystemslab.github.io/digital-planning-data-schemas/${jsonSchema["$id"]}/schema.json```

This is a workflow we should definitely keep talking about though! Schema releases and volume of apps are infrequent enough for now that the manual copy step should be manageable.

Reply:
Ah nice to have such a simple solution! And yep, agree there's fine-tuning on the release process to be done for sure in future.